go/extractor/extractor.go

```diff
@@ -478,6 +478,8 @@ func extractObjects(tw *trap.Writer, scope *types.Scope, scopeLabel trap.Label)
 			// Populate type parameter parents for named types.
 			if typeNameObj, ok := obj.(*types.TypeName); ok {
 				if tp, ok := typeNameObj.Type().(*types.Named); ok && !typeNameObj.IsAlias() {
+					// `typeNameObj` can only be an alias in versions of Go
+					// before 1.24.
```

Comment:
Should this be above the condition? It might also be useful to include what the implications of this are

Reply:
I have moved it and tried to make it clearer.